packages/integration/package.json

```diff
@@ -18,7 +18,7 @@
     "@babel/plugin-syntax-typescript": "^7.20.0",
     "@vanilla-extract/babel-plugin-debug-ids": "^1.0.4",
     "@vanilla-extract/css": "^1.14.0",
-    "esbuild": "0.17.6",
+    "esbuild": "npm:esbuild@~0.17.6 || ~0.18.0 || ~0.19.0",
```

Comment:
This syntax is not officially supported by npm, but I created a new project to [test the change](https://github.com/vanilla-extract-css/vanilla-extract/actions/runs/7649781260) and it worked.

<details><summary>Details</summary>

I tested with

```
  "dependencies": {
    "@vanilla-extract/css": "^1.14.0",
    "@vanilla-extract/esbuild-plugin": "^0.0.0-proxy-pr-1252-20240125041511",
    "esbuild": "^0.19.0"
  }
```

and

```
  "dependencies": {
    "@vanilla-extract/css": "^1.14.0",
    "@vanilla-extract/esbuild-plugin": "^0.0.0-proxy-pr-1252-20240125041511",
    "esbuild": "^0.17.6"
  }
```

</details>

Reply:
Update: it's [supported](https://github.com/npm/rfcs/blob/main/implemented/0001-package-aliases.md), but [not documented](https://github.com/npm/cli/issues/6839).